lua/entities/turret_40mm_frag/init.lua

```diff
@@ -13,7 +13,7 @@ self.Entity:SetModel( "models/items/ar2_grenade.mdl" )
 self.Entity:PhysicsInit( SOLID_VPHYSICS )      -- Make us work with physics,  	
 self.Entity:SetMoveType( MOVETYPE_NONE )   --after all, gmod is a physics  	
 self.Entity:SetSolid( SOLID_VPHYSICS )        -- CHEESECAKE!    >:3           
-self.Entity:SetColor(255,255,255,255)
+self.Entity:SetColor( Color( 255,255,255,255) )

```

Comment:
`Color` objects are technically expensive to create (as are `Vector` and `Angle`s), it's good practice to define them once and use them elsewhere.